src/style.css

```diff
@@ -0,0 +1,106 @@
+body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  color: var(--primary-text-color);
+
+  --accent-color: #00acdc;
+  --primary-text-color: #060b35;
+  --secondary-text-color: #616070;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  max-width: 100%;
+  height: auto;
+  margin-bottom: 40px;
+}
```

Comment:
You image is bigger, than in task, you need to set size for it and center it with `margin: 0 auto 40px;`, for example

Reply:
Ok!